app/controllers/admin/court_applications_controller.rb

```diff
@@ -13,6 +37,9 @@ def show
 
     def edit
       @court_application = CourtApplication.find(params[:id])
+      prosecution_case = ProsecutionCase.find(@court_application.prosecution_case)
+      @defendants = prosecution_case.defendants
+      @hearings = prosecution_case.hearings
```

Comment:
Q: just checking, does the `prosecution_case` always exist? are there cases where the `find` returns nil ?